### DISCUSSION TITLE
Raw user presets, needs clarification

### DISCUSSION THREAD

**Discussion by artincube** [photoprism/photoprism#3529] [Q&A]
When raws are converted to DNG from LightroomC for example, these keep some development parameters within the files.
Are these used?
Or only darktable presets?

Shall I enable user presets for raws if my only desire is to have them converted to jpegs using their embedded settings?

**Comment by lastzero** [comment]
At this time, PhotoPrism cannot use Adobe software to convert RAW images:

https://docs.photoprism.app/developer-guide/media/raw/#adobe-photoshop-lightroom

Most RAW converters allow you to save the parameters used to create JPEGs from RAW files in XMP sidecar files. They also have global settings to set default values.

That said, Adobe, RawTherapee, and Darktable unfortunately use different parameters and XMP namespaces for this (with very few exceptions).

Also note that to use existing RawTherapee and/or Darktable presets, this option must be enabled:

https://docs.photoprism.app/user-guide/settings/advanced/#use-presets
